tv_apps/fling-player/locales/fling-player.en-US.properties

```diff
@@ -1 +1 @@
-playing-video-from-device=Playing video casted from device
+playing-video-from-device=Playing video casted from {{ device }}
```

Comment:
1. Change your l10n key if you changed the content of string. L10n team require to know what has changed by this.
2. Uplifting l10n string changes need to be addressed when uplifting.